### PR TITLE
Kinto intro v2

### PR DESCRIPTION
**Pull request by almet**

### CODE REVIEW

**Comment by Natim** [comment]
LGTM

**Comment by almet** [comment]
I'll let you publish.

**Comment by Natim** [comment]
Yes sir: http://www.servicedenuages.fr/en/what-can-kinto-do-for-you

File: content/2015.12.what-can-kinto-do.rst [hunk 1]
```diff
@@ -23,14 +23,121 @@ Here is an attempt to fix this.
...
+That's useful if you want to synchronise data between devices, and also between
+users. You have the data on one location, and you want it in a different one.
+
+With Kinto.js, this is as simple as calling `.sync()` on the Kinto instance.
+
+.. code-block:: js
+
+   const tasks = new Kinto({
+       remote: "https://kinto.dev.mozaws.net/v1"
+   }).collection("tasks");
```
Comment (n1k0) [inline content/2015.12.what-can-kinto-do.rst:63]: As far as I can tell, this snippet doesn't call `sync()` at all.

Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst:63]: Yes we should fix that :)

Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst:63]: We can add:

``````
tasks
  .create({label: "First item", done: false})
  .then(_ => tasks.sync());```
``````

**Comment by n1k0** [comment]
Great write up otherwise!

File: content/2015.12.what-can-kinto-do.rst (hunk 1 above)
Comment (Natim) [inline content/2015.12.what-can-kinto-do.rst:63]: Here you go: http://www.servicedenuages.fr/en/what-can-kinto-do-for-you#synchronise-your-data

Comment (n1k0) [inline content/2015.12.what-can-kinto-do.rst:63]:
``` js
tasks
  .create({label: "First item", done: false})
  .then(_ => tasks.sync())
  .then(({ok}) => console.log(ok ? "Synced!" : "Woops!"))
  .catch(err => console.error(err));
```

I wish we could use generators :cry: 

Just for the sake of making you cringe:

``` js
try {
  yield tasks.create({label: "First item", done: false});
  const {ok} = yield tasks.sync();
  console.log(ok ? "Yay" : "Nay");
} catch (err) {
  console.error(err);
}
```

Comment (n1k0) [inline content/2015.12.what-can-kinto-do.rst:63]: > Here you go: http://www.servicedenuages.fr/en/what-can-kinto-do-for-you#synchronise-your-data

Small indentation issue

**Pull request merged**